fastlane/lib/fastlane/plugin_manager.rb

```diff
@@ -40,56 +48,65 @@ def add_dependency(plugin_name)
 
     # Makes sure, the user's Gemfile actually loads the Plugins file
     def plugins_attached?
-      gemfile_content.include?(code_to_attach)
+      gemfile_path && gemfile_content.include?(code_to_attach)
     end
 
     def ensure_plugins_attached!
       return true if plugins_attached?
-      UI.important("Looks like fastlane plugins are not yet set up for this project")
+      UI.important("It looks like fastlane plugins are not yet set up for this project.")
+
+      path_to_gemfile = gemfile_path || DEFAULT_GEMFILE_PATH
+
       if gemfile_content.to_s.length > 0
-        UI.important("fastlane will modify your existing Gemfile at path '#{gemfile_path}'")
+        UI.important("fastlane will modify your existing Gemfile at path '#{path_to_gemfile}'")
       else
-        UI.important("fastlane will create a new Gemfile at path '#{gemfile_path}'")
+        UI.important("fastlane will create a new Gemfile at path '#{path_to_gemfile}'")
       end
+
       UI.important("This change is neccessary for fastlane plugins to work")
 
-      if UI.confirm("Can fastlane modify the Gemfile at path '#{gemfile_path}' for you?")
-        attach_plugins!
-        UI.success("Successfully modified '#{gemfile_path}'")
+      if UI.confirm("Can fastlane modify the Gemfile at path '#{path_to_gemfile}' for you?")
+        attach_plugins!(path_to_gemfile)
+        UI.success("Successfully modified '#{path_to_gemfile}'")
       else
-        UI.important("Please add the following code to your Gemfile:")
+        UI.important("Please add the following code to '#{path_to_gemfile}':")
         puts ""
         puts code_to_attach.magenta # we use puts to make it easier to copy and paste
-        UI.user_error!("Please update your Gemfile and run fastlane again")
+        UI.user_error!("Please update '#{path_to_gemfile} and run fastlane again")
       end
       return true
     end
 
+    def create_gemfile(path)
```

Comment:
Maybe add a `!` operator, to be consistent with `attach_plugins!`, both methods change local files